CMakeLists.txt

```diff
@@ -25,14 +31,29 @@ include_directories(${SRC_INCLUDE_DIR})
 
 add_subdirectory(src)
 
-if (NUMBERS_TEST) 
-        message(STATUS "Building and running tests")
-        set(THIRD_PARTY_INCLUDE_DIR
-                ${PROJECT_SOURCE_DIR}/third_party
-        )
-        include_directories(${THIRD_PARTY_INCLUDE_DIR})
+if(NUMBERS_EXAMPLE)
+    add_subdirectory(examples)
+endif()
 
-        add_subdirectory(examples)
-        add_subdirectory(third_party)
-        add_subdirectory(tests)
+if(NUMBERS_TEST)
+    message(STATUS "Building and running tests")
+    set(THIRD_PARTY_INCLUDE_DIR
```

Comment:
We still need to learn how to configure the library so that it can be included by other libraries using `include(numbers)`.